docs/plugins/spectate.rst

```diff
@@ -2,16 +2,50 @@ spectate
 ========
 
 .. dfhack-tool::
-    :summary: Automatically follow exciting dwarves.
+    :summary: Automatically follow productive dwarves.
     :tags: fort interface
-    :no-command:
 
 Usage
 -----
 
 ::
 
     enable spectate
+    disable spectate
+    spectate <option> <value>
 
-The plugin will automatically switch which dwarf is being followed periodically,
-preferring dwarves on z-levels with the highest job activity.
+
+When enabled, the plugin will automatically switch which dwarf is being
+followed periodically, preferring dwarves on z-levels with the highest
+job activity.
+
+To set features that toggle between two states, use {0,1} to specify
+which state the feature should be in. Anything else will take any positive
+value.
+
+Changes to plugin settings will be saved per world. Whether the plugin itself
+is enabled or not is not saved.
```

Comment:
saving features/settings is a good idea, but should the plugin remember whether it was enabled? Would a player really want a fortress to start back up in spectate mode?